build/baremetalpi/boot/Makefile

```diff
@@ -0,0 +1,14 @@
+#
+# Download the firmware files to be required for boot (requires wget)
+#
+# These files must be copied along with the generated kernel.img
+#     onto a SD(HC) card with FAT file system.
+#
+
+firmware:
+	wget -q -O LICENCE.broadcom https://github.com/raspberrypi/firmware/blob/master/boot/LICENCE.broadcom?raw=true
+	wget -q -O bootcode.bin https://github.com/raspberrypi/firmware/blob/master/boot/bootcode.bin?raw=true
+	wget -q -O fixup.dat https://github.com/raspberrypi/firmware/blob/master/boot/fixup.dat?raw=true
+	wget -q -O start.elf https://github.com/raspberrypi/firmware/blob/master/boot/start.elf?raw=true
```

Comment:
Could likely use a git submodule for these? Unless the firmware is crazy large.

Reply:
it's 2 or 3 MB IIRC. This is the same Makefile they have on circle, I haven't inquired but i think there's some legal reason not to include them in other projects. Or maybe it's just to get the latest version? Anyway, i think this works ok, it's a one time task anyway :)